frontend/src/main/scala/bloop/engine/BuildLoader.scala

```diff
@@ -226,9 +224,13 @@ object BuildLoader {
         case Right(path) =>
           logger.debug(Feedback.configuredMetalsScalaProjects(projects))(DebugFilter.All)
           Some(path)
-        case Left(cause) =>
+        case Left(cause) if isSupportedVersion =>
           logger.displayWarningToUser(Feedback.failedMetalsScalaConfiguration(scalaVersion, cause))
           None
+
+        // We try to download anyway, but don't print the exception
+        case _ =>
```

Comment:
Can you explain what will happen if downloading semanticDB fails? Is this possible that user won't have code navigation and no indication why?

Also, I think comment might have better description, as this action (downloading) already took place and we handling failed case. For instance:
```scala
        // We tried to download semanticDB but scala version is unsupported - don't warn the user about it.
        case Left(cause) =>
          logger.debug(cause) (or sth along these lines?)
          None
```

Reply:
> Can you explain what will happen if downloading semanticDB fails? Is this possible that user won't have code navigation and no indication why?

No, they will always get the information that semanticdb options are missing. I decided not to show it to the user as a notification if the version is not expected to be supported. I could change that, but this should ideally not happen.

Reply:
I will update it when doing another change.